src/applications/proxy-rewrite/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Wasn't sure if this was an `application`, but it needs to be consumable by an external party, could go through versions, has a name, etc...

If this works well enough, we're going to want to pass it off to teamsite properties that we don't have  a proxy in front of for manual integration.

Reply:
Yeah, it almost seems like something that belongs in its own repo, but we aren't there yet.

If it matters at all, since in this case we aren't worried about wiring up to content pages or routing, I think we only need to define the `entryFile` and `appName`